umh-core/integration/docker_test_helpers.go

```
@@ -0,0 +1,93 @@
+func printContainerLogs() {
+	out, err := runDockerCommand("logs", containerName)
+	if err != nil {
+		fmt.Printf("Failed to get container logs: %v\n", err)
+		return
+	}
+	fmt.Printf("Container logs:\n%s\n", out)
+}
+
+// StopContainer stops and removes your container
+func StopContainer() {
+	runDockerCommand("stop", containerName)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)